### PR TITLE
AST-3428 - Container Layouts Revamp Compatibility

### PR DESCRIPTION
**Pull request by neerajmasai**
**Description** 

- Compatibility with Astra Revamped Layouts.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-3428

### Types of changes

- Bug Fix, breaking changes.

### How has this been tested?

- Should show the New Layout options
- Should show the Content Style options
- Should show the Sidebar Style options
- For New users - Layouts should apply accordingly.
- For Old users - Already set old layouts - should migrate to new options after updating plugin in editor and frontend.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by yssambare12** [comment]
@neerajmasai Please add a changelog.

**Comment by neerajmasai** [comment]
> @neerajmasai Please add a changelog.

There is no existing changelog file, should I add a new one @imnavanath ?

**Comment by imnavanath** [comment]
> > @neerajmasai Please add a changelog.
> 
> There is no existing changelog file, should I add a new one @imnavanath ?

@neerajmasai you can add in readme.txt file, you can see old changelogs over there.

**Comment by neerajmasai** [comment]
Added Changelog

**Review by yssambare12 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -61,6 +61,9 @@ Astra Bulk Edit plugin can be used only with the Astra theme.
 
 ## Changelog ##
 
+### 2.0.0 ###
```
Comment (imnavanath) [inline README.md:64]:
```suggestion
### 1.2.7 ###
```

Comment (imnavanath) [reply README.md:64]: No need to version it 2.0.0, as we are just adding compatibility for theme's major release.

File: astra-bulk-edit.php [hunk 2]
```diff
@@ -19,7 +19,7 @@
 /**
  * Set constants.
  */
-define( 'ASTRA_BLK_VER', '1.2.6' );
+define( 'ASTRA_BLK_VER', '2.0.0' );
```
Comment (imnavanath) [inline astra-bulk-edit.php:22]:
```suggestion
define( 'ASTRA_BLK_VER', '1.2.7' );
```

File: readme.txt [hunk 3]
```diff
@@ -61,6 +61,9 @@ Astra Bulk Edit plugin can be used only with the Astra theme.
 
 == Changelog ==
 
+= 2.0.0 =
```
Comment (imnavanath) [inline readme.txt:64]:
```suggestion
= 1.2.7 =
```

File: classes/class-astra-blk-meta-boxes-bulk-edit.php [hunk 4]
```diff
@@ -629,6 +664,55 @@ public function enqueue_admin_scripts_and_styles() {
 				);
 			}
 		}
+
+		/**
+		 * Migrate existing layouts to revamped layout combinations.
+		 *
+		 * @param  string $old_meta_content_layout Old Meta Layout.
+		 * @param  mixed  $meta Meta Options.
+		 * @return mixed $meta Meta Options.
```
Comment (imnavanath) [inline classes/class-astra-blk-meta-boxes-bulk-edit.php:673]:
```suggestion
		 * @return mixed $meta Meta Options.
		 * @since 1.2.7
		 *
```

**Comment by neerajmasai** [comment]
@imnavanath 

Two Main Cases Covered Here:
- Old Users with Astra Version < 4.2.0 will see old options, after updating Astra to >= 4.2.0 they will see new options and old options values will migrate to new one.
- New Users with Astra Version > 4.2.0 will see new options.

If needed can add a screencast, please let me know thanks!

File: astra-bulk-edit.php [hunk 5]
```diff
@@ -19,7 +19,7 @@
 /**
  * Set constants.
  */
-define( 'ASTRA_BLK_VER', '1.2.6' );
+define( 'ASTRA_BLK_VER', '1.2.7' );
```
Comment (imnavanath) [inline astra-bulk-edit.php:22]: Update here also version - https://d.pr/i/soBJiB

File: readme.txt [hunk 6]
```diff
@@ -61,6 +61,9 @@ Astra Bulk Edit plugin can be used only with the Astra theme.
 
 == Changelog ==
 
+= 1.2.7 =
+- Improvement: Compatibility with the Latest Astra Revamped Layout Options.
+
```
Comment (imnavanath) [inline readme.txt:66]: Update versioning in same file - https://d.pr/i/7Ywpzt

**Review by imnavanath — APPROVED** [review]

**Pull request merged**